test/check-ssl-hsts-preloadable_spec.rb

```diff
@@ -19,11 +19,11 @@
     expect { check.run }.to raise_error SystemExit
   end
 
-  it 'should pass check if the domain is preloadedable but has warnings' do
-    check.config[:domain] = 'oskuro.net'
-    expect(check).to receive(:warning).and_raise SystemExit
-    expect { check.run }.to raise_error SystemExit
-  end
+  #  it 'should pass check if the domain is preloadedable but has warnings' do
```

Comment:
if you are going to comment these out lets include a explanation as to why

Reply:
Can we leave these commented? I'm still not 100% sure what's required here.

Reply:
addressed in new commit